lib/src/meta.dart

```diff
@@ -99,3 +99,69 @@ void _forEachGeomInGeometryObject(
     throw Exception('Unknown Geometry Type');
   }
 }
+
+/// Callback for propEach
+typedef PropEachCallback = dynamic Function(
+    Map<String, dynamic>? currentProperties, num featureIndex);
+
+/// Iterate over properties in any [geoJSON] object, calling [callback] on each
+/// iteration. Similar to Array.forEach()
+///
+/// For example:
+///
+/// ```dart
+/// FeatureCollection featureCollection = FeatureCollection(
+///   features: [
+///     point1,
+///     point2,
+///     point3,
+///   ],
+/// );
+/// propEach(featureCollection, (currentProperties, featureIndex) {
+///   someOperationOnEachProperty(currentProperties);
+/// });
+/// ```
+void propEach(GeoJSONObject geoJSON, PropEachCallback callback) {
+  if (geoJSON is FeatureCollection) {
+    for (var i = 0; i < geoJSON.features.length; i++) {
+      if (callback(geoJSON.features[i].properties, i) == false) break;
+    }
+  } else if (geoJSON is Feature) {
+    callback(geoJSON.properties!, 0);
```

Comment:
I think this `!` bang can be dropped from properties

Reply:
Oh, I missed that